cajita/unit_test/tstArray3d.hpp

```diff
@@ -390,6 +390,20 @@ void arrayOpTest()
                   k < owned_space.max( Dim::K ); ++k )
                 for ( long l = 0; l < owned_space.extent( 3 ); ++l )
                     EXPECT_EQ( host_view( i, j, k, l ), 0.5 );
+
+    // Do a 3 vector update.
+    ArrayOp::assign( *array, 1.0, Ghost() );
+    ArrayOp::scale( *array, scales, Ghost() );
+    ArrayOp::assign( *array_2, 0.5, Ghost() );
+    ArrayOp::assign( *array_3, 1.5, Ghost() );
+    ArrayOp::update( *array, 3.0, *array_2, 2.0, *array_3, 4.0, Ghost() );
+    Kokkos::deep_copy( host_view, array->view() );
+    for ( long i = 0; i < ghosted_space.extent( Dim::I ); ++i )
+        for ( long j = 0; j < ghosted_space.extent( Dim::J ); ++j )
+            for ( long k = 0; k < ghosted_space.extent( Dim::K ); ++k )
+                for ( long l = 0; l < ghosted_space.extent( 3 ); ++l )
+                    EXPECT_EQ( host_view( i, j, k, l ),
+                               3.0 * scales[l] + 1.0 + 6.0 );
```

Comment:
Since `array_2` and `array_3` are constant, this test could miss indexing errors in the update.

Reply:
good call - updated